cql3/selection/selection.cc

```diff
@@ -442,7 +442,7 @@ class selection_with_processing : public selection {
 };
 
 // Return a list of columns that "SELECT *" should show - these are all
-// columns hidden ones (e.g. "virtual columns" used in materialized views).
+// columns except hidden ones (e.g. internal or view_virtual).
```

Comment:
I think it will be nicer to give the reader more concrete examples like in the old comment like "virtual columns" used in materialized views or "internal columns" used by LWT. This can give the user a bit of context. I don't know if anyone will know what "internal or view_virtual" refers to.